climada/util/plot.py

```diff
@@ -77,8 +81,9 @@ def geo_bin_from_array(array_sub, geo_coord, var_name, title,
     title : str or list(str)
         subplot title. If one provided, the same is used for all subplots.
         Otherwise provide as many as subplots in array_sub.
-    pop_name : bool, optional
-        add names of the populated places, by default True
+    pop_name : bool or str, optional
+        add names of the populated places, by default True. If set to 'markers' only the points
+        will be shown, without a description.
```

Comment:
Probably it is better then to change to ,None', 'names', 'markers' or something along these lines. Mixing types in variables is not a very good idea most of the time.